packages/core/lib/utils/string.dart

```diff
@@ -14,6 +14,9 @@ extension StringExt on String {
   }
 }
 
+String camelCase(String s) => s.camelCase();
```

Comment:
Doesn't seem to be an improvement compare to the original usages.

Reply:
Because now you can pass camelCase as a function, eg.

````
UniqueAssetType(assetType: assetType, style: camelCase)
```